simulation/traffic_simulator/src/entity/ego_entity.cpp

```diff
@@ -161,6 +164,9 @@ EgoEntity::EgoEntity(
   vehicle_model_type_(getVehicleModelType()),
   vehicle_model_ptr_(makeSimulationModel(vehicle_model_type_, step_time, parameters))
 {
+  // deactivate non-Ego jobs
+  // TODO: this job should be activate but needs data obtained from Autoware in BehaviorParameter
+  job_list_.inactivate(traffic_simulator::job::Type::OUT_OF_RANGE_DYNAMIC_CONSTRAINS);
```

Comment:
Why inactivate?

Reply:
[Here](https://tier4.atlassian.net/browse/AJD-696?focusedCommentId=117435) i commented:

> Should Performance (from the scenario) be taken into account when setting DynamicConstraints for Ego? (last column in the table)
> 
> I encountered a problem with [fixed](https://github.com/tier4/scenario_simulator_v2/blob/c425a60edc321fcfd56d4404ec42f0a64865d3e4/simulation/traffic_simulator/src/entity/ego_entity.cpp#L178-L188) max_acceleration==0 for Ego so I made some changes to ensure the effect of the Performance values. ([commit](https://github.com/tier4/scenario_simulator_v2/pull/979/commits/54f5abce604da9d679e207a82fe99671a4fb8964))

You answered:

> Why do you remove auto EgoEntity::getBehaviorParameter() const -> traffic_simulator_msgs::msg::BehaviorParameter , [...]?
> I think default value should be const.

I answered:

> I'm not sure why there were assignments like max acceleration=0 in EgoEntity::getBehaviorParameter() - without commenting out [these 2 lines](https://github.com/tier4/scenario_simulator_v2/blob/c425a60edc321fcfd56d4404ec42f0a64865d3e4/simulation/traffic_simulator/src/entity/ego_entity.cpp#L185-L186), out_of_range would not allow Ego to move.
> 
> However, this method still returned msgs::BehaviourParameter - not taking Performance into account. That's why I’ve removed this method and the definition from [VehicleEntity](https://github.com/tier4/scenario_simulator_v2/blob/54f5abce604da9d679e207a82fe99671a4fb8964/simulation/traffic_simulator/src/entity/vehicle_entity.cpp#L74-L77) is used.

You answered:

> This line [( entity/ego_entity.cpp#L181-L187)](https://github.com/tier4/scenario_simulator_v2/blob/f9357649bce1b2990574086398061d3afe2cba37/simulation/traffic_simulator/src/entity/ego_entity.cpp#L181-L187) shows that the part is still WIP and should not be changed without checking and considering the circumstances of why this is happening.
> I don't think it is a good implementation to use a base type function without inputting data obtained from Autoware in BehaviorParameter in the first place.

I answered:

> I see, thank you. I will undo the changes from this [commit](https://github.com/tier4/scenario_simulator_v2/commit/8215ade14ca5ad3cfa3c025f2d2fb5797884f150). 
> However, in this case, the job out_of_range for Ego can’t be run - I understand that it should only be run for non-Ego Vehicles?

So I've reverted `EgoEntity::getBehaviorParameter()` to the previous version and deactivated the `out_of_range` job for Ego - because there is a fixed value of `max_acceleration=0`.